test/presubmit-tests.sh

```diff
@@ -116,7 +129,7 @@ ui-build(){
 
 ### presubmit hooks ###
 
-run_build_tests() {
+build_tests() {
```

Comment:
Any reason we are changing the name of the function

Reply:
yes build test job in ci needs the func to be `build_tests` other wise it will run all func